proto/anyvalue.proto

```diff
@@ -23,6 +23,7 @@ syntax = "proto3";
 package opamp.proto;
```

Comment:
_should be as a different issue but I find it very related to this PR_ 

should it not be

```suggestion
package opentelemetry.opamp.proto.v1;
```

to follow our existing patterns in https://github.com/open-telemetry/opentelemetry-proto?

Reply:
That about 

```suggestion
package opamp.proto.v1;
```

?

Reply:
Yeah, that's fine, we can do that.

Reply:
Unless you need it for C# I suggest we don't touch it in this PR.